kubectl-plugin/waiops-status.md

```diff
@@ -6,180 +6,177 @@ https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/
 
 ## Highlights
 
-### Main status checker
+### Installation status checker
 ```
 $ oc waiops status
 
 ______________________________________________________________
 Installation instances:
 
-NAMESPACE   NAME                 PHASE     LICENSE    STORAGECLASS   STORAGECLASSLARGEBLOCK   AGE
-katamari    aiops-installation   Running   Accepted   rook-cephfs    rook-cephfs              14h
+NAMESPACE   NAME                  PHASE     LICENSE    STORAGECLASS   STORAGECLASSLARGEBLOCK   AGE
+cp4waiops   ibm-cp-watson-aiops   Running   Accepted   rook-cephfs    rook-ceph-block          99m
 ______________________________________________________________
 IRCore and AIOpsAnalyticsOrchestrator instances:
 
-KIND                         NAMESPACE   NAME    VERSION   STATUS
-IssueResolutionCore          katamari    aiops   3.3.0     Ready
-AIOpsAnalyticsOrchestrator   katamari    aiops   3.2.0     Ready
+KIND                         NAMESPACE   NAME    VERSION   STATUS   MESSAGE
+IssueResolutionCore          cp4waiops   aiops   3.3.0     Ready    All Services Ready
+AIOpsAnalyticsOrchestrator   cp4waiops   aiops   3.2.0     Ready    All Services Ready
```

Comment:
Is this analytics orchestrator expected to be back versioned?

Reply:
Yes, `AIOpsAnalyticsOrchestrator` is supposed to be 3.2.x.

https://ibm.ent.box.com/notes/932522810361?s=ldzag7hisnxlud69xdy4k1fec2641yvx

I grabbed this example output from a non-dev prod install instance.